client-participation/Dockerfile

```diff
@@ -6,6 +6,7 @@ WORKDIR /app
 RUN apk add --no-cache --virtual .build \
   g++ git make python
 
+RUN npm config set unsafe-perm true
```

Comment:
@patcon What's this command do? What makes it necessary?

Reply:
it's something to do with the docker filesystem. required whenever someone installs an npm package globally on alpine fs based images, apparently. not sure specifics